### ISSUE TITLE
Handle more complexe string with toRaiseError 

### ISSUE THREAD

**Issue by philibea** [tstyche/tstyche#34]
Using `toRaiseError` with a complexe error can be complexe and show an error 
```
Error: An argument for 'target' must be of type 'string | number'.
```

Example of code
```
toRaiseError(
    `Argument of type '"hello"' is not assignable to parameter of type '"doe" | "describe"'`,
  )
```

pnpm: v8.10.5
node: v20.10.0 
typescript: 5.3.2

**Comment by mrazauskas** [comment]
Should be fixed in [`1.0.0-beta.5`](https://github.com/tstyche/tstyche/releases/tag/v1.0.0-beta.5).

**Comment by mrazauskas** [comment]
Ups.. I just noticed that it still does not work as expected. Fix is on the way. Sorry.
